content-repo/extra-docs/articles/trust-custom-certificates-engines.md

```diff
@@ -0,0 +1,17 @@
+---
+id: trust-custom-certificates-engines
+title: Configure Engines to Trust Custom Certificates
+---
```

Comment:
Let's add a description to appear in the [articles index](https://pull-request-555--demisto-content-docs.netlify.app/docs/reference/articles-index)
```suggestion
title: Configure Engines to Trust Custom Certificates
description: Some description...
---
```